src/setup/askForRemoteUrl/askForRemoteUrl.ts

```diff
@@ -0,0 +1,20 @@
+import inquirer from 'inquirer';
+
+export const validateUrl = (input: string): boolean => {
+  const ans =
+    input.includes(':') && input.indexOf(':') == input.lastIndexOf(':');
+  const urlRegex = /^(https?|ftp):\/\/[^\s/$.?#].[^\s]*$/;
+  return ans && urlRegex.test(input);
```

Comment:
This not necessary, setup already asks for you to `Enter your talawa-api endpoint` using the `REACT_APP_TALAWA_URL`  value.